python/lsst/pipe/supertask/activator.py

```diff
@@ -421,18 +470,20 @@ def parse_and_run(cls):
 
         if args.list_tasks:
             print("\n List of available tasks: \n")
-            list_tasks = cls.get_tasks()
+            list_tasks,list_subclasses = cls.get_tasks()
             if len(list_tasks) == 0:
                 print('No Tasks in these packages: ')
                 for package in TASK_PACKAGES.keys():
                     print(package)
-            for task in list_tasks:
-                print(task)
+            for j in range(len(list_tasks)):
+                print('{0: <{zeros}}  :{1}'.format(list_tasks[j],list_subclasses[j],
+                zeros=len(max(list_tasks, key=len))))
+                #print(list_tasks[j],' ',list_subclasses[j])
             print()
             sys.exit()
 
         if args.list_packages:
-            print("\n List of available packages see by this activator: \n")
+            print("\n List of available packages seen by this activator: \n")
```

Comment:
I mean that "\n List" is different from "\nList" in that the first is indented after the new line.